hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/BlockManagerImpl.java

```diff
@@ -108,9 +108,11 @@ public long putBlock(Container container, BlockData data) throws IOException {
         // transaction is reapplied in the ContainerStateMachine on restart.
         // It also implies that the given block must already exist in the db.
         // just log and return
-        LOG.warn("blockCommitSequenceId {} in the Container Db is greater than"
-            + " the supplied value {}. Ignoring it",
-            containerBCSId, bcsId);
+        if (LOG.isDebugEnabled()) {
```

Comment:
Is it necessary to wrap a debug log entry in the `if (LOG.isDebugEnabled())` statement? It is my understanding, that so long as you are not doing string interpolation in the log message and are passing simple variables as any parameters to the log message (ie not doing any computation to calculate the values being passed), then LOG4J does the correct thing and there is no performance penalty to removing the IF wrapper.

Reply:
http://www.slf4j.org/faq.html#logging_performance suggests the IF wrapper is not needed for performance reasons.

Reply:
yepp, you are right, muscle memory I guess :D Thank you for checking it, I pushed a fix.